src/Controls/src/Core/Page/Page.iOS.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Microsoft.Maui.Controls.PlatformConfiguration;
+using UIKit;
+using Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific;
+
+namespace Microsoft.Maui.Controls;
+
+public partial class Page
+{
+	static void MapUpdateSafeAreaInsets(IViewHandler handler, IView view)
+	{
+		if (view is Page page && handler.PlatformView is UIView uiView)
+		{
+			var insets = uiView.SafeAreaInsets.ToThickness();
+			page.On<iOS>().SetSafeAreaInsets(insets);
+		}
+	}
+}
+
+public static class UIEdgeInsetsExtension
```

Comment:
Can you move this static class to its own file? Also, usually the naming scheme is something like `UIEdgeInsetsExtensions` (plural, in case we add more).

Reply:
For sure!

Reply:
@hartez  moved!